pyiron_atomistics/atomistics/structure/factory.py

```diff
@@ -62,6 +64,10 @@ def ase(self):
     def aimsgb(self):
         return self._aimsgb
 
+    @property
+    def laves(self):
```

Comment:
I'm not sure if the nesting only for the three laves phases makes sense, if we don't want them poluting the `structure` namespace maybe `compound` or `binary` is more general.

Reply:
Yeah, I was hoping for some critical feedback here! I had been thinking `extra.laves` in the long run, but I also don't mind `compound`. I would like to avoid `binary` because I'm hoping that the C36 method will allow mixing species on one of the sublattices (for the benefit of the SFB).

Reply:
After sleeping on it, I really like `compound` and made a commit for it with both of us as authors. Now I can move the B2 and D03 stuff here from the FeAl project as well, which is nice! 

I'm going to leave the discussion here unresolved though, in case there is an even better term than `compound` floating around that neither of us have thought of.

Reply:
Jup, compound works I think.